src/Uno.UI.RemoteControl.Host/Startup.cs

```diff
@@ -22,7 +22,7 @@ internal class Startup
 
 		public IConfiguration Configuration { get; }
 
-		public void Configure(IApplicationBuilder app, IOptionsMonitor<RemoteControlOptions> optionsAccessor, IHostingEnvironment env)
+		public void Configure(IApplicationBuilder app, IOptionsMonitor<RemoteControlOptions> optionsAccessor)
```

Comment:
Why ?

Reply:
The parameter was not used and it caused warning as the type is deprecated and replaced with something else (I think `IHostEnvironment`)